doc/ref_cert/lfn/chapters/chapter09.md

```diff
@@ -19,11 +19,47 @@ Define, and describe the purpose of this chapter to be:
 <a name="9.2"></a>
 ## 9.2 RM/RA-1 Requirements
 
-- List out reference to Test Cases Requirements (coming from Chapter 3) and map them to real project test scripts.
+According to [RC1 Chapter04](https://github.com/cntt-n/CNTT/blob/master/doc/ref_cert/lfn/chapters/chapter04.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`

Reply:
https://cntt-n.github.io/ returns 404.
Not a big deal to switch to relative path if it's a global rule here. I'm diving into it.

Reply:
Absolute doesn't work in both. If a person viewing the md files, it breaks.